src/psyclone/tests/transformations_test.py

```diff
@@ -221,3 +219,126 @@ def test_regiontrans_wrong_children():
         RegionTrans.validate(rtrans, parent.children)
     assert ("Cannot apply a transformation to multiple nodes when one or more "
             "is a Schedule" in str(err.value))
+
+
+def test_regiontrans_wrong_options():
+    '''Check that the validate method raises the expected error if passed
+        options that are not contained in a dictionary.
+
+    '''
+    # RegionTrans is abstract so use a concrete sub-class
+    region_trans = ACCParallelTrans()
+    with pytest.raises(TransformationError) as excinfo:
+        RegionTrans.validate(region_trans, None, options="invalid")
+    assert ("Transformation apply method options argument must be a "
+            "dictionary but found 'str'." in str(excinfo.value))
+
+# Tests for ProfileRegionTrans
+
+
+def dummy_nodes():
+    '''Utility routine which create a minimal invoke and schedule
+    hierarchy for use in subsequent ProfileRegionTrans tests.
+
+    :returns: a list containing a single node connected to a minimal \
+    InvokeSchedule which is connected to a minimal Schedule. This \
```

Comment:
I don't think this is right - you don't have an InvokeSchedule connected to a Schedule? Also, please indent this text slightly.

Reply:
Well spotted.